docs/src/test-webserver-js.md

```diff
@@ -18,6 +18,8 @@ export default defineConfig({
     command: 'npm run start',
     url: 'http://127.0.0.1:3000',
     reuseExistingServer: !process.env.CI,
+    stdout: 'ignore',
```

Comment:
this is part of https://github.com/microsoft/playwright/pull/23474 - so probably should be reverted from this PR